chain/client/src/debug.rs

```diff
@@ -163,8 +163,8 @@ impl Handler<WithSpanContext<DebugStatus>> for ClientActor {
             DebugStatus::EpochInfo => {
                 Ok(DebugStatusResponse::EpochInfo(self.get_recent_epoch_info()?))
             }
-            DebugStatus::BlockStatus => {
-                Ok(DebugStatusResponse::BlockStatus(self.get_last_blocks_info()?))
+            DebugStatus::BlockStatus(height) => {
```

Comment:
yay!! 

(this will be great for archival nodes  - as it will allow us to easily look at the historical data)